src/core/server/logging/logging_config.ts

```diff
@@ -85,7 +86,7 @@ export const config = {
   }),
 };
 
-export type LoggingConfigType = Omit<TypeOf<typeof config.schema>, 'appenders'> & {
+export type LoggingConfigType = Pick<TypeOf<typeof config.schema>, 'loggers' | 'root'> & {
```

Comment:
I recall being advised against using `Pick` for types because it makes the API documentation less helpful. I think the advice was to rather duplicate types.

Reply:
Yea... OTOH these types are already based on `TypeOf<typeof schema>`, which are probably also more than broken in the API documentation right now. Removing such references is probably a higher-level effort for Core's codebase.